src/test/java/api/support/builders/InstanceBuilder.java

```diff
@@ -74,6 +74,15 @@ public InstanceBuilder withId(UUID id) {
       this.identifiers);
   }
 
+  public InstanceBuilder withTitle(String title) {
+    return new InstanceBuilder(
+      this.id,
+      title,
+      this.contributors,
+      this.instanceTypeId,
+      this.identifiers);
+  }
```

Comment:
Why is this needed rather than the constructor parameter?

Reply:
Removed this method, it's been unused. I intended to use it for building Instance object but eventually went with one of the existing constructors.